config/opensearch_dashboards.yml

```diff
@@ -327,6 +327,10 @@
 # "all": The data source can be managed by all users. Default to "all".
 # data_source_management.manageableBy: "all"
 
+# Set the backend roles in groups, whoever has the backend roles defined in this config will be regard as dataSourceAdmin.
+# DataSource Admin will have the access to all the data source saved objects inside OpenSearch Dashboards by api.
+# data_source_management.dataSourceAdmin: []
```

Comment:
```suggestion
# data_source_management.dataSourceAdmin.groups: ["data_source_management"]
```
Should we follow the pattern of dashboard admin?

Reply:
updated